core/modules/script.py

```diff
@@ -65,6 +117,55 @@ class ScriptLeafSkel(BaseScriptAbstractSkel):
         vfunc=lambda value: not FILE_PATTERN.match(value),
     )
 
+    globalid = StringBone(
+        descr="Global identifyer",
+        vfunc=lambda value: not FILE_PATTERN.match(value),
+        # unique=True,
+        params={
+            "tooltip": "If the script got copied from an other scriptreository, you may want to set an "
+            "unique identifyer so you may match it for updates."
+                }
+    )
+
+    label = StringBone(
+        descr="Button label",
+        defaultValue="Run script",
+        )
+
+    tooltip = TextBone(
+        descr="Tooltip for button",
+        validHtml=None,
+        )
+
+    icon = SelectBone(
+        descr="Icon",
+        values=[icon.removesuffix(".svg") for icon in sorted(os.listdir("static/svg"))],
+        params={
+                "tooltip": "Icon which is used by /vi to display in the Buttonbar."
+            }
+        )
+
+    buttonbar = RecordBone(
+        descr="Visible in Buttonbar",
+        multiple=True,
+        using=ButtonbarSkel,
+        required=False,
+        format="$(module)",
+        )
+
+    userdescr = TextBone(descr="Description for user")
+    devdescr = TextBone(descr="Description for developers ")
+
+    globalaccess = SelectBone(
+        descr="Required access rights of specific modules",
+        values=lambda: {
+
+            right: i18n.translate("server.modules.user.accessright.%s" % right, defaultText=right)
+            for right in sorted(conf["viur.accessRights"])
+        },
+        multiple=True,
+    )
```

Comment:
Respect existing style guide!!

You should probably think of putting this stuff into a `RecordBone(multiple=False)`, to make distinction of the script file and the action config.

```suggestion
    label = StringBone(
        descr="Label",
        defaultValue="Run script",
    )

    tooltip = TextBone(
        descr="Tooltip",
        validHtml=None,
    )

    icon = SelectBone(
        descr="Icon",
        values=[icon.removesuffix(".svg") for icon in sorted(os.listdir("static/svg"))],
        params={
            "tooltip": "Icon which is used by /vi to display in the Buttonbar."
        }
    )

    actions = RecordBone(
        descr="Provides actions",
        multiple=True,
        using=ScriptLeafActionSkel,
        required=False,
        format="$(module)",
    )

    userdescr = TextBone(
        descr="Description for user"
    )
    devdescr = TextBone(
        descr="Description for developers "
    )

    globalaccess = SelectBone(
        descr="Required access rights of specific modules",
        values=lambda: {
            right: i18n.translate("server.modules.user.accessright.%s" % right, defaultText=right)
            for right in sorted(conf["viur.accessRights"])
        },
        multiple=True,
    )
```